src/components/ThirdPartyCollectionDetailPage/CollectionItem/CollectionItem.tsx

```diff
@@ -80,7 +94,11 @@ export default class CollectionItem extends React.PureComponent<Props> {
           <Grid.Column className={styles.column}>
             <div>{this.getTokenId()}</div>
           </Grid.Column>
-          <Grid.Column className={styles.column}>
+          <Grid.Column width={3} className={`${styles.column} ${styles.statusColumn} ${styles[status]}`}>
```

Comment:
Left this one as it was since it was used that way above too (using more than 1 class by just concatenating both strings). I prefer the `classnames` approach as well :)